cluster/ci/config/prow/config.yaml

```diff
@@ -168,6 +168,7 @@ tide:
     - openshift/dedicated-admin-operator
     - openshift/descheduler-operator
     - openshift/elasticsearch-operator
+    - openshift/enhancements
```

Comment:
Looking at the `branches` for this config, you've turned on merging for release branches but not `master`. Did you mean to?

Reply:
huh, I thought it would get the master branch from some other configs? I thought that other repo's in this config section got merge to master?

Reply:
This specific section does the release branches, the other does `master` -- look at the `branches` listed

Reply:
(it's in the same file, a couple hundred lines away)

Reply:
👍 sorry about that thanks for the once over